src/JsonTxBuilderBase.sol

```diff
@@ -51,4 +52,108 @@ abstract contract JsonTxBuilderBase is CommonBase {
+                string.concat("Self-destructed account: ", vm.toString(accountAccess.account))
+            );
+
+            for (uint256 j; j < accountAccess.storageAccesses.length; j++) {
+                Vm.StorageAccess memory storageAccess = accountAccess.storageAccesses[j];
+
+                if (!storageAccess.isWrite) continue; // Skip SLOADs.
+
+                uint256 value = uint256(storageAccess.newValue);
+                if (isLikelyAddressThatShouldHaveCode(value)) {
+                    // Log account, slot, and value if there is no code.
+                    string memory err = string.concat(
+                        "Likely address in storage has no code\n",
+                        "  account: ",
+                        vm.toString(storageAccess.account),
+                        "\n  slot:    ",
+                        vm.toString(storageAccess.slot),
+                        "\n  value:   ",
+                        vm.toString(bytes32(value))
+                    );
+                    require(address(uint160(value)).code.length != 0, err);
+                } else {
+                    // Log account, slot, and value if there is code.
+                    string memory err = string.concat(
+                        "Likely address in storage has unexpected code\n",
+                        "  account: ",
+                        vm.toString(storageAccess.account),
+                        "\n  slot:    ",
+                        vm.toString(storageAccess.slot),
+                        "\n  value:   ",
+                        vm.toString(bytes32(value))
+                    );
+                    require(address(uint160(value)).code.length == 0, err);
+                }
+
+                require(
+                    storageAccess.account.code.length != 0,
+                    string.concat("Storage account has no code: ", vm.toString(storageAccess.account))
+                );
+                require(
+                    !storageAccess.reverted,
+                    string.concat("Storage access reverted: ", vm.toString(storageAccess.account))
+                );
+                require(
+                    storageAccess.account.code.length != 0,
+                    string.concat("Storage account has no code: ", vm.toString(storageAccess.account))
+                );
+                require(
+                    !storageAccess.reverted,
+                    string.concat("Storage access reverted: ", vm.toString(storageAccess.account))
+                );
+            }
+        }
+    }
+
+    /// @notice Returns a list of addresses which are expected to be in storage, but will not to have code on this
+    ///         chain. Examples of such addresses include EOAs, predeploy addresses, and inbox addresses.
+    function getCodeExceptions() internal pure virtual returns (address[] memory) {
+        revert("getCodeExceptions not implemented");
```

Comment:
this will "trick" solidity into not emitting an unreachable code warning
```suggestion
        require(false, "getCodeExceptions not implemented");
```